packages/form-js-viewer/src/render/components/form-fields/Textfield.js

```diff
@@ -75,6 +75,17 @@ Textfield.config = {
   label: 'Text field',
   group: 'basic-input',
   emptyValue: '',
-  sanitizeValue: ({ value }) => (isArray(value) || isObject(value)) ? '' : String(value),
+  sanitizeValue: ({ value }) => {
+    if (isArray(value) || isObject(value)) {
+      return '';
+    }
+
+    // sanitize newlines to spaces
+    if (typeof value === 'string') {
+      return value.replace('\n', ' ');
```

Comment:
Is `sanitizeValue` called on change? I think we could change to the option below to cover all occurrences, specially if a user pastes the text.

Also, do we have to worry about other special symbols like tab? We could do a more complete replacement like this `value.replace(/\n/g, ' ');`

```suggestion
      return value.replace(/[\r\n\t]/g, ' ');
```

Reply:
Yeah I agree, let's do that.

Reply:
And not it's not called onChange, I think we want to handle that at the component level if we need to. Sometimes we make use of it onChange, but it's explicit.